src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -32,4 +34,14 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @PutMapping("/{id}")
+    public BookDto updateBook(@PathVariable Long id, @RequestBody CreateBookRequestDto requestDto) {
+        return bookService.updateById(id, requestDto);
+    }
+
+    @DeleteMapping("/{id}")
+    public void deleteBook(@PathVariable Long id) {
+        bookService.delete(id);
+    }
```

Comment:
add corresponding response statuses for each endpoint in controller, like `NO_CONTENT`, `CREATED`, `OK`